canonical/ubuntu.com
# perf: Merge print.scss into styles.scss to reduce the number of .css requests on page load

## Done

- Merge print.scss into styles.scss to reduce the number of .css requests on page load
- Removes custom styling for `u-no-print` as it now [exists in vanilla](https://vanillaframework.io/docs/utilities/no-print)

## QA

- Go to https://ubuntu.com/, load the page with network and see there is a separate request for print.css.
- Attempt to print the page and see that it is formatted for printing (no nav, images adjusted etc)
- Go to https://ubuntu-com-14213.demos.haus/ load the page with network and see there is **not** a separate request for print.css.
- Attempt to print the page and see that it is formatted the same as on live

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14184
